classes/classes/PlayerAppearance.as

```diff
@@ -145,8 +145,11 @@ package classes
 					outputText("  You have a cat-like face, complete with moist nose and whiskers.  Your " + player.skinDesc + " is " + player.furColor + ", hiding your " + player.skin(true,false) + " underneath.", false);
 				if (player.hasScales()) 
 					outputText("  Your facial structure blends humanoid features with those of a cat.  A moist nose and whiskers are included, but overlaid with glittering " + player.skinFurScales() + ".", false);
-				if (player.eyeType != EYES_BLACK_EYES_SAND_TRAP) 
-					outputText("  Of course, no feline face would be complete without vertically slit eyes.");
+				if (player.eyeType != EYES_BLACK_EYES_SAND_TRAP)
+				{
+					outputText("  Of course, no feline face would be complete without vertically slit eyes");
+					outputText(!player.hasReptileEyes() ? "." : ", although they come with a second set of eyelids, which is somewhat unusual for a cats face.");
+				}
 			}
```

Comment:
That part was somewhat nasty. I guess adding EYES_CAT later would make things easier in the future.
Adding an eyeCount to the player vars, adding hasSpiderEyes() to Creature.as (Similar to isTaur()) and deprecating EYES_FOUR_SPIDER_EYES would be an idea, too. So cats can still have 4 vertically slitted eyes ^^